packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1273,6 +1275,45 @@ describe('shallow', () => {
         expect(wrapper.find(Component.displayName)).to.have.lengthOf(2);
       });
     });
+
+    describeIf(is('>= 16.6'), 'Suspense & lazy', () => {
+      it('finds Suspense and its children when no lazy component', () => {
+        class Component extends React.Component {
+          render() {
+            return (
+              <div>test</div>
+            );
+          }
+        }
+        const SuspenseComponent = () => (
+          <Suspense fallback={false}>
+            <Component />
+          </Suspense>
+        );
+
+        const wrapper = shallow(<SuspenseComponent />);
+
+        expect(wrapper.find('Suspense')).to.have.lengthOf(1);
+        expect(wrapper.find(Component)).to.have.lengthOf(1);
+      });
+
+      it('finds Lazy when given lazy component', () => {
+        const LazyComponent = lazy(() => import('./_helpers/dynamicImportedComponent.jsx'));
+        const Fallback = () => <div />;
+        const SuspenseComponent = () => (
+          <Suspense fallback={<Fallback />}>
+            <LazyComponent />
+          </Suspense>
+        );
+
+        const wrapper = shallow(<SuspenseComponent />);
+
+        expect(wrapper.find('Suspense')).to.have.lengthOf(1);
+        expect(wrapper.find(LazyComponent)).to.have.lengthOf(1);
+        // won't render fallback in shallow renderer
```

Comment:
can we add another test that `.dive()`s the `Suspense`, and then becomes the fallback?

Reply:
Oops, I forgot to test shallow render `Suspense` directly, and I found out that for now `react-test-renderer/shallow` doesn't allow render `Suspese`:

```js
import ShallowRenderer from "react-test-renderer/shallow"

const renderer = new ShallowRenderer()
renderer.render(<Suspense fallback={false} />)

```

will throw:

```
Invariant Violation: ReactShallowRenderer render(): Shallow rendering works only with custom components, but the provided element type was `symbol`.
    at invariant (/Users/yishan/Documents/Projects/enzyme/packages/enzyme-adapter-react-16/node_modules/react-test-renderer/cjs/react-test-renderer-shallow.development.js:54:15)
    at ReactShallowRenderer.render (/Users/yishan/Documents/Projects/enzyme/packages/enzyme-adapter-react-16/node_modules/react-test-renderer/cjs/react-test-renderer-shallow.development.js:382:78)
```

It looks like that react shallow renderer cannot recoginze the `Suspense` type symbol. I should check this at first... So unless react-test-renderer supports this I think we can only support `Suspense` and `lazy` in mount. 😢